packages/opencensus-core/src/stats/view.ts

```diff
@@ -187,9 +196,58 @@ export class BaseView implements View {
     }
   }
 
+  /**
+   * Gets view`s metric
+   * @returns {Metric}
+   */
+  getMetric(): Metric {
+    const {type} = this.metricDescriptor;
+    let startTimestamp: Timestamp;
+
+    switch (type) {
+      case MetricDescriptorType.GAUGE_INT64:
+      case MetricDescriptorType.GAUGE_DOUBLE:
+        startTimestamp = null;
+        break;
+      default:
+        const [seconds, nanos] = process.hrtime();
+        startTimestamp = {seconds, nanos};
+    }
+
+    const timeseries: TimeSeries[] = [];
+
+    Object.keys(this.rows).forEach(key => {
+      const {tags} = this.rows[key];
+      const labelValues: LabelValue[] = MetricUtils.tagsToLabelValues(tags);
+      const point: Point = this.toPoint(startTimestamp, this.getSnapshot(tags));
+      timeseries.push({startTimestamp, labelValues, points: [point]});
+    });
+
+    return {descriptor: this.metricDescriptor, timeseries};
+  }
+
+  /**
+   * Converts snapshot to point
+   * @param timestamp The timestamp
+   * @param data The aggregated data
+   * @returns {Point}
+   */
+  private toPoint(timestamp: Timestamp, data: AggregationData): Point {
+    let value;
+    if (data.type === AggregationType.DISTRIBUTION) {
+      // TODO: Add examplar transition
+      value = Object.assign(
+          {bucketOptions: {explicit: {bounds: data.buckets}}}, data);
```

Comment:
I think this value should match with Point-> DistributionValue.

Please consider something like this... 
```
value = {
count: data.count, 
sum: data.sum, 
sumOfSquaredDeviation: data.sumOfSquaredDeviation, 
bucketOptions: {explicit: {bounds: data.buckets}}, 
buckets: data.bucketCounts
}
```

please add test for all these. Thank you.

Reply:
Done